Codegen/src/Generators/DefaultGenerator.cs

```diff
@@ -895,15 +903,19 @@ private static void AdvanceToNextStateAndAdjustTokenProperties(TokensLine tempTo
                     while ((t = scanner.GetNextToken()) != null)
                     {
                         if (t.TokenType == TokenType.GLOBAL_STORAGE)
-                        {//Skip GLOBAL-STORAGE SECTION. tokens
+                        {//Skip and remove GLOBAL-STORAGE SECTION. tokens by generating spaces instead.
+                            sw.Write(new System.String(' ', t.Text.Length));
                             while ((t = scanner.GetNextToken()) != null && t.TokenType != TokenType.PeriodSeparator)
                             {
+                                sw.Write(new System.String(' ', t.Text.Length));
                                 AdvanceToNextStateAndAdjustTokenProperties(tempTokensLine, t);
                             }
+                            if (t != null)
+                                sw.Write(new System.String(' ', t.Text.Length));
                             AdvanceToNextStateAndAdjustTokenProperties(tempTokensLine, t);
                             continue;
                         }
-                        else if (t.TokenType == TokenType.LevelNumber)
+                        else if (t.TokenType == TokenType.LevelNumber && base.Layout == ColumnsLayout.CobolReferenceFormat && t.Column > 6 && t.Column < 73)
```

Comment:
What about other formats ?

Reply:
The other format is FreeTextFormat and it begins at colum 1 with no unused columns.